flow/layers/layer.h

```diff
@@ -34,6 +34,20 @@
 
 namespace flow {
 
+// This should be an exact copy of the Clip enum in
+// flutter/package/flutter/lib/src/paining/basic_types.dart
```

Comment:
typo: should be `packages/flutter/lib/src/painting/basic_types.dart`

Reply:
We now moves the Clip enum to painting.dart.